common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -448,6 +447,7 @@ def definition_to_xml(self, resource_fs):
             'sub': self.sub,
             'download_track': json.dumps(self.download_track),
             'download_video': json.dumps(self.download_video),
+            'edx_video_id': self.edx_video_id,
```

Comment:
was this necessary to add?  The `edx_video_id` field was already exported with the module.